helm-chart/index.yaml

```diff
@@ -22,6 +22,31 @@
 apiVersion: v1
 entries:
   openj9-jitserver-chart:
+  - apiVersion: v2
+    appVersion: 0.26.0
+    created: "2021-03-12T18:49:10.914225842-08:00"
+    description: "Eclipse OpenJ9 JITServer Helm Chart. \n\nLicense\n\nThis chart is
```

Comment:
Why has the license message changed from the 0.24.0 which was read:
`By installing this product you accept the Eclipse Public License v2.0`

Reply:
The description has been updated in latest commit. 

The previous description was generated by `helm`, which is copied directly from `Chart.yaml`. 
https://github.com/eclipse-openj9/openj9-utils/blob/master/helm-chart/openj9-jitserver-chart/Chart.yaml#L25

Should we keep these two versions consistent? If yes, could you please confirm which version is desired? 
``` txt
(Chart.yaml)
Eclipse OpenJ9 JITServer Helm Chart.   

License 
This chart is made available under the terms of the Eclipse Public License 2.0
which accompanies this distribution and is available at https://www.eclipse.org/legal/epl-2.0/
or the Apache License, Version 2.0 which accompanies this distribution and
is available at https://www.apache.org/licenses/LICENSE-2.0.

The JDK binaries installed by this chart are licensed under the GPLv2+CE.
```

```txt
(index.yaml)
Eclipse OpenJ9 JITServer Helm Chart. 

License
By installing this product you accept the Eclipse Public License v2.0: 
https://www.eclipse.org/legal/epl-2.0/.
```

Reply:
I see your point now. The license from Chart.yaml seems more comprehensive, but what worries me is that different releases in `index.yaml` will have different license terms. I am ok with using the more comprehensive version if you update the text for 0.24 as well.

Reply:
I have updated the two descriptions in `index.yaml` to the more comprehensive version. Could you please check again? Thanks.